src/_data/people.yml

```diff
@@ -42,8 +42,3 @@ victor:
   github: victorwuky
   twitter: victorwuky
   active: true
-abdullah:
-  name: Abdullah Zameek
-  github: abdullahzameek
-  twitter: abdullahzameek
-  active: true
```

Comment:
actually, can you just change this to `false` and leave the user there?

that makes it easier for us to reference you in other places after you leave

Reply:
(see how Maria's is done)

Reply:
Ahhh I see, made that change!